.github/workflows/push.yml

```
@@ -51,12 +51,20 @@ jobs:
       - if: ${{ steps.cache-npm.outputs.cache-hit != 'true' }}
         name: List the state of node modules
         run: npm install
-      - run: npm run test -- --watchAll=false --coverage
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)